bootloader/slots/slot.cpp

```diff
@@ -37,6 +38,24 @@ const UserlandHeader* Slot::userlandHeader() const {
     Ion::Device::Flash::LockSlotA();
   }
 
+  // Erase the bootloader integrated in slots in Epsilon 20
+  if (m_userland2Header->isValid()) {
+		if (m_address == 0x90000000) {
+			// Check if bootloader is present in slot A
+			if (*(uint32_t*)0x90010000 != 0xFFFFFFFF) {
+        // Erase bootloader in slot A
+				Ion::Device::ExternalFlash::EraseSector(9);
+			}
+		}
+		else if (m_address == 0x90400000) {
+			// Check if bootloader is present in slot B
+			if (*(uint32_t*)0x90410000 != 0xFFFFFFFF) {
+        // Erase bootloader in slot B
+				Ion::Device::ExternalFlash::EraseSector(73);
```

Comment:
Fix comment indentation
```suggestion
			// Check if bootloader is present in slot A
			if (*(uint32_t*)0x90010000 != 0xFFFFFFFF) {
                // Erase bootloader in slot A
				Ion::Device::ExternalFlash::EraseSector(9);
			}
		}
		else if (m_address == 0x90400000) {
			// Check if bootloader is present in slot B
			if (*(uint32_t*)0x90410000 != 0xFFFFFFFF) {
                // Erase bootloader in slot B
				Ion::Device::ExternalFlash::EraseSector(73);
```